pkg/machinery/client/resolver/roundrobin.go

```diff
@@ -62,9 +64,21 @@ func (r *roundRobinResolver) start() error {
 	var addrs []resolver.Address //nolint:prealloc
 
 	for _, a := range strings.Split(r.target.Endpoint, ",") { //nolint:staticcheck
+		addr := net.FormatAddress(a)
```

Comment:
not totally clear why can't we prepare `endpoint:port` before we pass down to this function, so that we don't have to append ports here?